packages/relay/src/index.tsx

```diff
@@ -1,6 +1,3 @@
 export { connectionUpdater, connectionDeleteEdgeUpdater } from './mutationUtils';
 
-// TODO - consume these 3 functions directly from react-relay
-export { useSubscription } from './useSubscription';
-export { loadQuery } from './loadQuery';
-export { useMutation } from './useMutation';
+// TODO - consume these 3 functions directly from react-relay
```

Comment:
remove TODO

this change will break relay workshop

you need to find all usages of useSubscription, loadQuery and useMutation 

like this

import { useMutation } from '@workshop/relay'

and rename it to

import { useMutation } from 'react-relay'

make sure you do at least 1 single test for files that you changed